artifacts/etcdstorage/crd.yaml

```diff
@@ -44,4 +44,16 @@ spec:
                     pattern: '^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$'
                   namespace:
                     type: string
-                    pattern: '^[a-z0-9]([-a-z0-9]*[a-z0-9])?$'
\ No newline at end of file
+                    pattern: '^[a-z0-9]([-a-z0-9]*[a-z0-9])?$'
+            signingCertificateValidityMinutes:
+              type: integer
+              minimum: 10
+              maximum: 43830
+            servingCertificateValidityMinutes:
+              type: integer
+              minimum: 10
+              maximum: 43830
+            clientCertificateValidityMinutes:
+              type: integer
```

Comment:
we also have a `time.Duration` wrapper in apimachinery. Then you can write 10m and 0.5y.